website/contributing/release-branch-cut-and-rc0.md

```diff
@@ -1,27 +1,30 @@
 ---
-id: release-candidate-minor
-title: Release Minor Candidate
+id: release-branch-cut-and-rc0
```

Comment:
Great improvement :)

Reply:
ahhaah - yeah I mean I felt that I needed to change the name a bit to reflect the change in title